src/getter.rs

```diff
@@ -509,4 +509,17 @@ impl Getter for CppCode {
 
 impl Getter for PreprocCode {}
 impl Getter for CcommentCode {}
-impl Getter for JavaCode {}
+impl Getter for JavaCode {
+    fn get_space_kind(node: &Node) -> SpaceKind {
+        use Java::*;
+
+        let typ = node.object().kind_id();
+        match typ.into() {
+            ClassDeclaration => SpaceKind::Class,
```

Comment:
Maybe you could add InterfaceDeclaration here too.
Maybe it's worthing adding a SpaceKind::Interface because an interface contributes to loc for example but it expected to have other metrics equal to 0 since it just some functions declarations.
So having interface in the output will help the user the understand this numbers and maybe (probably ?) skip this data since they aren't really useful (but maybe they're).

Reply:
I mapped `InterfaceDeclaration` as a `SpaceKind::Class` and it didn't show up at all in the hierarchy.

With AND without gave:

unit: foo.java
- function: print
- class: A6
-- function: print
-- function: main 

for the following code file:

```java
package com.java;

interface printable{  
  void print();  
}  

class A6 implements printable{  
  public void print(){System.out.println("Hello");}  
    
  public static void main(String args[]){  
    A6 obj = new A6();  
    obj.print();  
  }  
} 
```

Adding a `SpaceKind::Interface` interface could be interesting if it indeed showed up in the hierarchy. I would expect:

unit: foo.java
- interface:  printable
-- function: print
- class: A6
-- function: print
-- function: main 

Although as you said, not sure how many metrics interfaces actually contribute. Evaluating with "Lay of least surprise" I do think the `SpaceKind::Interface` makes sense but that should be a separate PR I would think?